config/defaults.ini

```diff
@@ -88,6 +89,7 @@ packages_add =
   json-glib-tools
   jq
   libfsapfs-utils
+  librsvg2-bin
```

Comment:
I don't know that the explanation in the commit message is true, but I do know that without it my test build failed, and I am running a new build before leaving for the day to test this solution…

Reply:
Sadly, that did not address the issue.

```
+ 16:30:07 INFO 70-flatpak-appstream-catalog: Composing AppStream XML: appstream-compose --origin=eoscustom-master-amd64-amd64.231110-162702.base --prefix=/var/cache/eos-image-builder/tmp/ostree-co/ostree/deploy/eos/var/lib/flatpak/exports --output-dir=/var/cache/eos-image-builder/tmp/out --icons-dir=/var/cache/eos-image-builder/tmp/appstream-compose-icons org.freedesktop.Bustle
Processing application org.freedesktop.Bustle
Error loading desktop file: Failed to load icon: image format was not recognized
```

I'm pretty sure it is an SVG thing

Reply:
It helps to install the correct package, which I could have cribbed from the eos-ostree-builder commit I am taking inspiration from.